src/BasePresenter.php

```diff
@@ -71,6 +71,22 @@ public function getRouteKeyName()
     }
 
     /**
+     * Retrieve model for route model binding.
+     *
+     * @param mixed $routeKey
+     *
+     * @return mixed
+     */
+    public function resolveRouteBinding($routeKey)
+    {
+        if (method_exists($this->wrappedObject, 'resolveRouteBinding')) {
```

Comment:
`is_callable([$this->wrappedObject, 'resolveRouteBinding'])` would be more correct, since `method_exists` returns `true` for private methods.